zetasql/parser/bison_parser.y

```diff
@@ -3721,6 +3724,14 @@ show_statement:
       }
     ;
 
+flush_statement:
+    "FLUSH" "PRIVILEGES"
+    {
+        $$ = MAKE_NODE(ASTFlushStatement, @$, {"PRIVILEGES"});
```

Comment:
You need add ASTFlushStatement yourself.

And its not necessary to pass a "privilege" string as constructor argument